life/about-me-instructions.md

```diff
@@ -0,0 +1,24 @@
+# About Me Sections
+
+I'm a big fan of self-care, and I think part of that involves recognizing that you can still learn something about yourself and acknowledging your happies and crappies. This section is specifically for those kinds of TILs.
+
+I have a separate branch on this for my own personal use that at least for the time being, I don't plan on pushing. That said, I wanted to add this because I'm getting a lot out of it and wanted to share the idea.
+
+Basically you call the file `<your-github-name>-about-me.md` in the `life` and treat it like a rolling issue, so you add something you learn about yourself in maybe list-form or diary entry-type posts. The beauty of it is, it doesn't fucking matter how you format this because this is for *you*. *You* decide if you want to push the branch or not. Maybe you find out you did or said something gross and you want to get better about it, and having it documented helps you work on it. Maybe you find out you have mono and you want to tell people to stay away! Who knows! I personally don't need to push my branch up, but the point is, it's *your* decision.
+
+Anyway, if you like examples, I've been doing bulleted lists based on topics:
+```md
+## Personality
+  - I apparently don't like crowds that much!
+
+## Habits
+  - I play with my hair if I'm nervous!
+
+## Gender/Sexuality
+  - omg I think I'm bisexual?
+
+## Health (use if you find something you might want to check out)
+  - My stomach hurts if I eat kale!
+```
+
+These files are for self-care, self-love, and self-exploration. Learn from yourself and use that to grow as a person and avoid negativity. - [@laurpaik](https://github.com/laurpaik) ❤️
```

Comment:
idk, it didn't in the Markdown Preview on Atom, so I assumed I had to link it manually